fastlane/Fastfile

```diff
@@ -4,6 +4,12 @@ import_from_git(url: 'https://github.com/rakutentech/ios-buildconfig.git')
 default_platform(:ios)
 
 platform :ios do
+  desc "Project update"
+  lane :updatePods do |options|
```

Comment:
Could you update `DEV.md` with information that this repo uses submodules that need to be initialized?

Reply:
I've updated the doc, tell me what you think